core/document/src/operation.rs

```diff
@@ -39,4 +40,10 @@ pub enum Operation {
 	AddFolder {
 		path: Vec<LayerId>,
 	},
+	MountTempFolder {
+		path: Vec<LayerId>,
+	},
+	DiscardTempFolder,
```

Comment:
Should we call this `UnmountTempFolder`?

Reply:
yeah we can split the mount/unmount/clear functionality

Reply:
I would expect that unmount would also clear (next time you mount, it will start fresh).